pkg/hostpath/hostpath.go

```diff
@@ -339,7 +350,12 @@ func mapBackingResourceToPod(hpv *hostPathVolume) error {
 			return err
 		}
 		upsertRangerCM := func(key, value interface{}) bool {
-			return commonUpsertRanger(podConfigMapsPath, hpv.SharedDataKey, key, value)
+			cm, _ := value.(*corev1.ConfigMap)
+			payload := Payload{
+				StringData: cm.Data,
+				ByteData:   cm.BinaryData,
+			}
+			return commonUpsertRanger(podConfigMapsPath, hpv.SharedDataKey, key, payload)
```

Comment:
Here you could handle any errors returned by the commonUpsertRanger function and return true/false and log the error.

Reply:
Yeah this will capture at least errors on initial creations.  If we get errors on subsequent updates after the Pod's volume has been provisioned and the configmap/secrets have been updated, this path won't help.

For that, I'm going to add k8s event recording to at least augment the the error logging in the pod.